guides/catalog/02 Creating and Managing Products.ipynb

```diff
@@ -39,98 +39,129 @@
-   "id": "afa69bb7-7442-4d4d-9466-a49d130e5bfd",
+   "id": "f29f859f-b93f-4dc9-9503-f5ee54297b59",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "org = dl.auth.Auth().payload[\"org\"]\n",
-    "org"
+    "user_hash = dl.auth.Auth().namespace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1829e262-a26e-494f-a9e2-65dc03ef7c26",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "pid = f\"sample-rgb-product:{user_hash}\"\n",
+    "pid"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "a5155b5d-14a8-4071-bc3a-7f00a514157e",
+   "id": "6ca0deff-3be9-4a64-bbde-22a894925de8",
    "metadata": {},
    "source": [
-    "Since you may have other users within your organization who have run this same example, let's append our user ID to our product ID.\n",
-    "\n",
-    "Each Descartes Labs account has a unique user ID:"
+    "We can create our new dataset by calling [`Product.get_or_create()`](https://docs.descarteslabs.com/descarteslabs/catalog/docs/product.html#descarteslabs.catalog.Product.get_or_create), giving it a name, [`tags`](https://docs.descarteslabs.com/descarteslabs/catalog/docs/product.html#descarteslabs.catalog.Product.tags), and importantly saving everything through [`Product.save()`](https://docs.descarteslabs.com/descarteslabs/catalog/docs/product.html#descarteslabs.catalog.Product.save):"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "6a28f311-aed9-4f87-905f-b954b353423b",
+   "id": "a56aa3ec-019a-44f2-8a3a-179b6bdba424",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "user_id = dl.auth.Auth().namespace\n",
-    "user_id"
+    "product = Product.get_or_create(pid)\n",
+    "product.name = \"Example RGB Product\"\n",
+    "product.tags = [\"examples\"]\n",
+    "product.save()\n",
+    "product"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "b631be2e-3f1e-45eb-9956-02471f0f0c2d",
-   "metadata": {},
+   "id": "bf22f30e-1f3e-4912-9b38-cc546ab62172",
+   "metadata": {
+    "tags": []
+   },
    "source": [
-    "#### **_Note on Product Creation:_** \n",
-    "We do not always need to append our user ID to the product ID as in the following cell. This notebook is designed for demonstration purposes where other users within your organization may have already ran this example.\n",
+    "#### Finding your Product's Namespace\n",
+    "\n",
+    "Note the product ID in the above output. This prepended string is the new product's _namespace_. You can retrieve that namespace by the [`Auth`](https://docs.descarteslabs.com/descarteslabs/auth/readme.html) API, defaulting to the user's organization:\n",
+    "\n",
+    "    my-org-id:my-product-id\n",
     "\n",
-    "In practice, as long as your product has a **unique** ID within your organization you can call it whatever you'd like! "
+    "or, for self-signup and trial users, their unique user hash:\n",
```

Comment:
I don't think that "self-signup" is a term that a new user would necessarily understand to include themselves. Not sure of a better term though.

Reply:
good point, I will change the wording up! like
"or,` if you don't have an organization, their unique user hash:"